errors/apiErrors.ts

```diff
@@ -10,11 +10,38 @@ import { i18n } from '../utils/lang';
 import { throwError } from './standardErrors';
 import { HubSpotAuthError } from '../models/HubSpotAuthError';
 import { HttpMethod } from '../types/Api';
+import {
+  HubSpotHttpError,
+  isHubSpotHttpError,
+} from '../models/HubSpotHttpError';
 
 const i18nKey = 'errors.apiErrors';
 
+function extractHttpErrorFields(error: unknown): {
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  data: any;
+  status?: number;
+  code?: string;
+} {
+  if (isHubSpotHttpError(error)) {
+    return {
+      data: error.data,
+      status: error.status,
+      code: error.code,
+    };
+  }
+  if (isAxiosError(error)) {
```

Comment:
Can you clarify why we still need this line if all calls are using the new Axios interceptor that will convert the errors?

Reply:
We do not need this anymore, I was leaving it around for backwards compatibility, but we can remove it and update the CLI simultaneously